src/Sylius/Behat/Context/Ui/Admin/ProductCreationContext.php

```diff
@@ -14,7 +14,7 @@
 namespace Sylius\Behat\Context\Ui\Admin;
 
 use Behat\Behat\Context\Context;
-use Sylius\Behat\Page\Admin\Product\CreateSimpleProductPageInterface;
+use Sylius\Behat\Page\Admin\Product\SimpleProduct\CreateSimpleProductPageInterface;
```

Comment:
I'm not sure if this additional directory is needed 🤔

Reply:
I reverted these changes 👍